inc/saisamplepacket.h

```diff
@@ -47,6 +47,25 @@ typedef enum _sai_samplepacket_type_t
 } sai_samplepacket_type_t;
 
 /**
+  * @brief SAI mode of samplepacket sampling
+  */
+typedef enum _sai_samplepacket_mode_t
+{
+    /** A profile defining sample packet attributes. 
+      * The profile can be configured on a port, in which case, 
+      * an actual session is instantiated which is private for that port. 
+      * In case multiple ports are configured with the same profile, 
+      * sampling is independently done per port. */
+    SAI_SAMPLEPACKET_MODE_EXCLUSIVE,
+    
+    /** An instance of sample packet session. 
+      * This session can be attached to multiple ACL entries. Traffic matching 
```

Comment:
By its definition, the mode is not bound to acl rule, it can also be applied to the port. the same is true for the exclusive mode. Can you make this clear in the comments?